### ISSUE TITLE
No mention of max_bin parameter in Python API

### ISSUE THREAD

**Issue by xhlulu** [microsoft/LightGBM#2236]
## Version

Latest

## Description

In the [Parameter tuning section](https://lightgbm.readthedocs.io/en/latest/Parameters-Tuning.html?highlight=max_bin#for-faster-speed) of the docs, it is recommended to tune `max_bin` as a tradeoff between accuracy and speed:

> For Faster Speed
> * Use bagging by setting bagging_fraction and bagging_freq
> * Use feature sub-sampling by setting feature_fraction
> * Use small max_bin
> * Use save_binary to speed up data loading in future learning
> * Use parallel learning, refer to Parallel Learning Guide
> 
> For Better Accuracy
> * Use large max_bin (may be slower)
> * Use small learning_rate with large num_iterations
> * Use large num_leaves (may cause over-fitting)
> * Use bigger training data
> * Try dart

However, there is no mention of the `max_bin` parameter in both the Scikit-Learn and Training API.

Was the parameter removed? Thanks!

**Comment by StrikerRUS** [comment]
If there is no corresponding argument for any parameter from this page https://lightgbm.readthedocs.io/en/latest/Parameters.html, then you can pass it via `params` dict in training API or via `kwargs` in sklearn wrapper.
For instance:
```
lgb.train({'max_bin': 50}, train_set)

lgb.LGBMClassifier(max_bin=50)
```
